tbtaf/publisher/report_ai.py

```diff
@@ -0,0 +1,34 @@
+# publisher/ai_analyzer.py
+
+import requests
+import json
+
+# Constantes para la configuración de Ollama
+OLLAMA_API_URL = "http://localhost:11434/api/generate"
```

Comment:
Lets load this from an env variable, then if not defined, then we should error out.